DESCRIPTION

```diff
@@ -1,18 +1,23 @@
 Package: PKPDsim
 Type: Package
-Title: Simulate ODE models
+Title: Simulate ODE Models
 Version: 1.0.124
-Date: 2021-06-08
-Author: Ron Keizer
-Maintainer: Ron Keizer <ron@insight-rx.com>
+Date: 2021-12-06
+Authors@R: c(
+        person("Ron", "Keizer", email = "ron@insight-rx.com", role = c("aut", "cre")),
+        person("Jasmine", "Hughes", email = "jasmine@insight-rx.com", role = "aut"),
+        person("Dominic", "Tong", email = "dominic@insight-rx.com", role = "aut"),
+        person("Kara", "Woo", email = "kara@insight-rx.com", role = c("aut")),
+        person("InsightRX", role = c("cph", "fnd")))
 Depends: R (>= 3.0.2)
-Imports: Rcpp (>= 0.12.9), BH, data.table, lubridate, stringr, MASS,
```

Comment:
Removing BH from Imports and moving it to LinkingTo generates a NOTE on CI, but having it the other way around generates a NOTE locally. I believe this is due to CI using an older version of Rcpp and that it's correct to move it to LinkingTo.

Reply:
Argh I was wrong, this is actually an error. We might need to update the irx-r-base docker image to use newer package versions, at least for Rcpp.

Reply:
Updating the Rcpp version doesn't seem to fix it, so I am reverting this change for now and will look into it separately.